apps/api/test/TestManager.ts

```diff
@@ -0,0 +1,63 @@
+import { INestApplication, ValidationPipe } from "@nestjs/common";
+import { ConfigService } from "@nestjs/config";
+import { Test } from "@nestjs/testing";
+import RedisStore from "connect-redis";
+import * as session from "express-session";
+import * as passport from "passport";
+import { createClient } from "redis";
+import { AppModule } from "../src/app.module";
+
+export class TestManager {
+    // biome-ignore lint/suspicious/noExplicitAny: it is any.
+    public httpServer: any;
+    public app: INestApplication;
```

Comment:
`INestApplication<NestExpressApplication>` could be more expressive. not mandatory.